js/binance.js

```diff
@@ -6546,7 +6546,7 @@ module.exports = class binance extends Exchange {
             }
             let signature = undefined;
             if (this.secret.indexOf ('PRIVATE KEY') > -1) {
-                signature = this.rsa (query, this.secret);
+                signature = this.encodeURIComponent (this.rsa (query, this.secret));
```

Comment:
@pietrodn Hum, does not seem to be working on my side, when creating an order:

```
ccxt git:(fix/binance-rsa) ✗ n binance createOrder "LTC/USDT" market sell 0.2
Debugger listening on ws://127.0.0.1:50107/260dc9d4-6987-48f0-b3a2-4322da83f68c
For help, see: https://nodejs.org/en/docs/inspector
Debugger attached.
2023-01-26T14:34:00.580Z
Node.js: v19.0.1
CCXT v2.6.90
binance.createOrder (LTC/USDT, market, sell, 0.2)
AuthenticationError binance {"code":-2015,"msg":"Invalid API-key, IP, or permissions for action."}
---------------------------------------------------
[AuthenticationError] binance {"code":-2015,"msg":"Invalid API-key, IP, or permissions for action."}

    at throwExactlyMatchedException  js/base/Exchange.js:2419              throw new exact[string] (message);                                              
    at handleErrors                  js/binance.js:6638                    this.throwExactlyMatchedException (this.exceptions['exact'], error, feedback);  
    at                               js/base/Exchange.js:603               const skipFurtherErrorHandling = this.handleErrors (response.status, response.s…
    at processTicksAndRejections     node:internal/process/task_queues:95                                                                                  
    at async timeout                 js/base/functions/time.js:181         return await Promise.race ([promise, expires.then (() => { throw new TimedOut (…
    at fetch2                        js/base/Exchange.js:2042              return await this.fetch (request['url'], request['method'], request['headers'],…
    at request                       js/binance.js:6667                    const response = await this.fetch2 (path, api, method, params, headers, body, c…
    at createOrder                   js/binance.js:3623                    const response = await this[method] (this.extend (request, requestParams));     
    at async run                     examples/js/cli.js:293                const result = await exchange[methodName] (... args)                            

binance {"code":-2015,"msg":"Invalid API-key, IP, or permissions for action."}
```

Reply:
@carlosmiei does `master` work for you?

Reply:
@pietrodn It also doesn't work for order creation

Reply:
```
npm run cli.js binance createOrder "LTC/USDT" market sell 0.2
```
works for me (and works also in master). What doesn't work in `master` and works on this branch is any authenticated operation on `binanceusdm`.